CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # bedrock-vue-wallet ChangeLog
 
+## 19.0.0 -
+
+### Changed
+- **BREAKING**: `WalletLayout.vue` no longer ensures credential handler is installed on account change.
+
+### Added
+- `InstallHandlerBtn.vue` now installs the credential handler.
```

Comment:
Fix vue component file name to match component name -- and alphabetize `Added` to be above `Changed`.

```suggestion
### Added
- `InstallHandlerButton.vue` now installs the credential handler.
```